README.md

```diff
@@ -61,7 +62,7 @@ DC_BOT_PERMISSIONS=274878131264
 ## :rocket: How to run the bot
 
 ```sh
-npm run serve
+npm run dev
```

Comment:
I didn't notice this error, thank you for fixing it! ❤️

Reply:
Np!